mesonbuild/interpreter.py

```diff
@@ -448,6 +448,7 @@ def partial_dependency_method(self, args, kwargs):
         return DependencyHolder(self.held_object.get_partial_dependency(**kwargs))
 
 class GeneratorHolder(InterpreterObject, ObjectHolder):
+    @FeatureNewKwargs('generator', '0.43.0', 'capture')
```

Comment:
@jon-turney added a commit in the other PR that makes this an array: https://github.com/mesonbuild/meson/pull/3661/commits/fb7168635dc551139fd2bae20e750cfce9e06416

 Is that not correct?

Reply:
True, that is correct, but I removed everything from build.py to interpreter.py. I'll fix that right after.